### ISSUE TITLE
In CSS files "../" is replaced with "" (empty string)

### ISSUE THREAD

**Issue by inez** [ember-cli/ember-cli-preprocess-registry#51]
Seems that this issue was introduced sometimes between v3.1.2 and v3.2.1, but most likely v3.2.1 exactly, this commit: https://github.com/ember-cli/ember-cli-preprocess-registry/commit/59aeb88ef6b9f4580018bf2d97440fe063753125

I will work on polishing this bug/issue report, but that's all I've got for now (going to sleep, 1:23 am here).

**Comment by Emrvb** [comment]
It's caused by a new version of clean-css (4.x.x) that ships with the new broccoli-clean-css (2.x.x)  from a7d2eeba3edf06d3639bf54833a3398c6b91cd84

Quick work around: update ember-cli-build.js accordingly

```
var app = new EmberApp(defaults,{
  minifyCSS: {
    enabled: true,
    options: {
      rebase: false
    }
  }
});
```

This will probably minify your test and development builds as well.

I'll create a pull request with an actual fix later today.

**Comment by Turbo87** [comment]
this was fixed by reverting to the old `clean-css` (https://github.com/ember-cli/ember-cli-preprocess-registry/pull/55). sorry about that 😞 

will release it in the next few minutes.

**Comment by Turbo87** [comment]
released as v3.2.2
